google-cloud-tools-plugin/resources/config.properties

```diff
@@ -15,4 +15,7 @@
 #
 
 # Place cloud tools plugin configuration here.
-feature.appengine.flex=true
\ No newline at end of file
+feature.appengine.flex=true
+
+# Should be used in any Notifications explicitly produced by the plugin.
+notifications.plugin.groupdisplayid=Google App Engine plugin
```

Comment:
Notifications requires a groupDisplayId String to group notifications for. That String will appear in Settings -> Notifications so you can mute/unmute/etc. notifications of that kind. I'm thinking that all notifications from the plugin should have the same groupDisplayId, and config.properties is the place where this config seemed to make more sense. i.e., there's a "# Place cloud tools plugin configuration here." comment right at the beginning of it.

Feel free to suggest a better place.